backend/models/WatchedEpisode.mjs

```diff
@@ -0,0 +1,14 @@
+import mongoose from 'mongoose';
+const { Schema, model } = mongoose;
+
+const schema = new Schema(
+  {
+    episodeId: { type: Number },
+  },
+  {
+    versionKey: false,
+    collection: 'watchedEpisodes',
+  }
+);
+
+export default model('WatchedEpisode', schema);
```

Comment:
I haven't actually seen this before with the `collection` 😅. @mbosselmann and I were wondering if this is set by intention or what your reasons are?

Usually the collection is defined by the model name but in plural. So if the models name is `Todo` then the collection name would be `todos`. Just wondering though 🙂

Reply:
I didn't know about the collection either, Coach Andrea helped me last night in the ask channel.
Oh you're right. Thanks for the hint. The flow was to blame, because i had set the collection for the watchlist.
I delete the collection in `WatchedEpisode.mjs` and `WatchedMovie.mjs`